src/Core.js

```diff
@@ -142,7 +172,7 @@ export const logout = async (conf) => {
   const fullResponse = await fetch(logoutUrl, options);
 
   if (fullResponse.ok) {
-    TokenStorage.clearTokens();
+    await TokenStorage.clearSession();
```

Comment:
Could we use return TokenStorage.ClearSession ?

Reply:
I'm not sure of this: the user is awaiting for the logout method to be completed, since it is `async`. If inside the method body we don't await another async method call, are you sure that this won't lead to potentially dangerous situations?